samples/ehr-flask/katanemo/integrations/flask_auth.py

```diff
@@ -0,0 +1,69 @@
+import json
+import katanemo_sdk
+import katanemo_auth_sdk
+import requests
+from flask import request as flask_request
+
+class KatanemoFlaskAuth:
+    
+    def __init__(self, app):
+        self.app = app
+        self.api_client = None
+        self.auth_client = None
+        self.api_access_token = None
+        self.client_id = None
+
+    def register(self, client_name, service_id, client_id, client_secret):
+        self.client_id = client_id
+        self.client_secret = client_secret
+        access_api_client = katanemo_sdk.AccessControlApi(self._get_access_api_client())
+        req = katanemo_sdk.OAuthTokenRequest(clientId=client_id, clientSecret=client_secret)
+        resp = access_api_client.get_short_term_token(req)
+        self.api_access_token = resp.access_token
+
+    def authorize_request(self, access_token, request_path, http_method, request_body):
```

Comment:
Is this /arc/authorize? How do we annotate this call so that its easier for developers to choose between this method and the OAuth2.0 /authorize method?

Reply:
yes this is making call to arc/authorize - I will rename it to arc_authorize_request so its more clear